### PR TITLE
Update the vendor for implement get prev region

### PR DESCRIPTION
**Pull request by liukun4515**
### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to update the `tidb-ansible` repository
 - Need to be included in the release note

### CODE REVIEW

**Comment by liukun4515** [comment]
PATL @arthurkiller .
I have updated the vendor for you.

File: Gopkg.lock [hunk 1]
```diff
@@ -38,65 +49,73 @@
...
 
 [[projects]]
+  digest = "1:973dbcbbb1be662b61604319582383c315add70906a68d14894f31542ffc3a25"
   name = "github.com/go-sql-driver/mysql"
   packages = ["."]
+  pruneopts = "NUT"
   revision = "3955978caca48c1658a4bb7a9c6a0f084e326af3"
 
 [[projects]]
-  name = "github.com/gogo/protobuf"
```
Comment (arthurkiller) [inline Gopkg.lock]: @liukun4515 You have delete this dep too

File: Gopkg.lock [hunk 2]
```diff
@@ -106,104 +125,123 @@
...
 [[projects]]
   branch = "master"
+  digest = "1:7d5f99346aa63d23681f4d92708469f32384d0c26722a4de5725bd0f22caedac"
   name = "github.com/ngaut/sync2"
   packages = ["."]
+  pruneopts = "NUT"
   revision = "7a24ed77b2efb460c1468b7dc917821c66e80e55"
 
 [[projects]]
-  name = "github.com/opentracing/basictracer-go"
```
Comment (arthurkiller) [inline Gopkg.lock]: and this

**Comment by arthurkiller** [comment]
@liukun4515 I did the same as you, but Pingcap said these dependencies should not be removed

**Comment by arthurkiller** [comment]
PTAL . lol

**Comment by liukun4515** [comment]
Please wait, I will report this thing, and reply as soon as possible.

**Comment by arthurkiller** [comment]
@liukun4515 thx a lot, and I will also report this.

**Comment by arthurkiller** [comment]
sure. Move this repo into real path, then make it

Best regards 
✉️✉️✉️✉️✉️✉️✉️✉️✉️✉️✉️✉️✉️✉️

Arthur lee

**Pull request closed without merging**